data/lib/core/game.lua

```diff
@@ -20,39 +20,26 @@ function Game.convertIpToString(ip)
 end
 
 function Game.getReverseDirection(direction)
-	if direction == WEST then
-		return EAST
-	elseif direction == EAST then
-		return WEST
-	elseif direction == NORTH then
-		return SOUTH
-	elseif direction == SOUTH then
-		return NORTH
-	elseif direction == NORTHWEST then
-		return SOUTHEAST
-	elseif direction == NORTHEAST then
-		return SOUTHWEST
-	elseif direction == SOUTHWEST then
-		return NORTHEAST
-	elseif direction == SOUTHEAST then
-		return NORTHWEST
-	end
-	return NORTH
+	local config = {
+		[DIRECTION_NORTH] = DIRECTION_SOUTH,
+		[DIRECTION_SOUTH] = DIRECTION_NORTH,
+		[DIRECTION_EAST] = DIRECTION_WEST,
+		[DIRECTION_WEST] = DIRECTION_EAST
+	}
+
+	return config[direction] or DIRECTION_SOUTH
```

Comment:
You completely ignored diagonal directions and changed the default from north to south.